### DISCUSSION TITLE
Offsite copies - how do you do it?

### DISCUSSION THREAD

**Discussion by JeffWDH** [abbbi/virtnbdbackup#182] [General]
For my offsite VM backup copies I've been using Borg backup (https://github.com/borgbackup/borg) against my backup directory. This does a pretty nice job of deduplicating the backup data.

What else are you guys using for offsite?

**Comment by abbbi** [comment]
at the moment i dont, but if so, i would probably go with a backup pool on zfs and use zfs send/ or zrep to sync the zfs snapshots to another remote location.
